packages/web-client/app/components/common/footer/index.hbs

```diff
@@ -0,0 +1,48 @@
+<footer class="cardstack-footer" ...attributes>
+  <div class="cardstack-footer__section cardstack-footer__section--top">
+    <div class="cardstack-footer__important-links">
+      <a href="https://support.cardstack.com" target="_blank" rel="noopener noreferrer" >
```

Comment:
Too bad Opera and Android WebView don't support it, but all major browsers [imply noopener](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#browser_compatibility) when `target="_blank"` is given. 

Additionally, it [looks that](https://html.spec.whatwg.org/multipage/links.html#link-type-noreferrer) `noreferrer` implies `noopener`.

Reply:
Yeah, this is soon becoming respecifying a browser default, but was thinking we could leave rel="noopener noreferrer" in until `ember-template-lint` decides that it's out of date.

Reply:
Yeah I agree. Hope it soon becomes archaic to add these attributes to external links. 😀